testing/tests/DevExpress.ui.widgets/speedDialAction.tests.js

```diff
@@ -339,4 +338,35 @@ QUnit.module("check action buttons click args", (hooks) => {
     });
 });
 
+QUnit.module("add visible option", (hooks) => {
+    hooks.afterEach(() => {
+        $("#fab-one").dxSpeedDialAction("instance").dispose();
+        $("#fab-two").dxSpeedDialAction("instance").dispose();
+    }),
+
+    test("check rendering", (assert) => {
+        const firstSDA = $("#fab-one").dxSpeedDialAction().dxSpeedDialAction("instance");
+
+        firstSDA.option("visible", false);
+
+        assert.equal($("." + FAB_CLASS).length, 0, "invisible if change visible option to false");
```

Comment:
I suggest to make some variable from "." + FAB_CLASS

Reply:
Variable will store the first value

Reply:
Not for length for ('.'+FAB_CLASS). This is const.